sklbench/benchmarks/sklearn_estimator.py

```diff
@@ -74,7 +74,7 @@ def get_estimator(library_name: str, estimator_name: str):
 def get_estimator_methods(bench_case: BenchCase) -> Dict[str, List[str]]:
     # default estimator methods
     estimator_methods = {
-        "training": ["fit"],
+        "training": ["partial_fit", "fit"],

```

Comment:
I think `partial_fit` should be explicitly requested in config since most of incremental estimators can work in both modes:
```suggestion
        "training": ["fit"],
```